### PR TITLE
add grep-like pattern searching to .ccignore

### PR DESCRIPTION
**Pull request by julie-is-late**

### CODE REVIEW

File: src/main/java/cceclipseplugin/core/CCIgnore.java [hunk 1]
```diff
@@ -99,17 +100,19 @@ public void loadCCIgnore(IProject p) {
...
 					
-					String path = Paths.get(line).normalize().toString();
-					
-					if (path.equals(""))
+					// ignore empty lines
+					if (line.equals(""))
 						continue;
 					
-					ignoredFiles.add(path);
+					System.out.println("ADDING TO CCIGNORE: " + line);
```
Comment (ObsessiveOrange) [inline src/main/java/cceclipseplugin/core/CCIgnore.java]: Delete this

File: src/main/java/cceclipseplugin/core/CCIgnore.java [hunk 2]
```diff
@@ -125,6 +128,10 @@ public void loadCCIgnore(IProject p) {
 	 * 
 	 */
 	private void serverCleanUp(IProject p) {
```
Comment (ObsessiveOrange) [inline src/main/java/cceclipseplugin/core/CCIgnore.java:129]: Does this even work?

**Review by ObsessiveOrange — APPROVED** [review]
Delete one debug statement

File: src/main/java/cceclipseplugin/core/CCIgnore.java (hunk 2 above)
Comment (julie-is-late) [reply src/main/java/cceclipseplugin/core/CCIgnore.java:129]: no, but this is getting opened in a different issue b/c it's out of scope for this

File: src/main/java/cceclipseplugin/core/CCIgnore.java (hunk 1 above)
Comment (julie-is-late) [reply src/main/java/cceclipseplugin/core/CCIgnore.java]: k

**Pull request merged**